### ISSUE TITLE
Embedded frame redirects 307 https://github.com/orgs/giscus/discussions/1298 for new script tag from giscus.app

### ISSUE THREAD

**Issue by niccokunzmann** [giscus/giscus#1356]
I copied the script from the giscus.app website.
When I embed the script, I can see this redirect.

```
307
location: https://github.com/orgs/giscus/discussions/1298
```

So, no comments appear but the discussion claims that I use the giscus repository. I wonder why because I do not use it.

![grafik](https://github.com/giscus/giscus/assets/564768/5a17b9e6-144d-4588-bf92-8d0ee87ca097)

Resulting script:

```html
<script src="https://giscus.app/client.js"
        data-repo="niccokunzmann/open-web-calendar"
        data-repo-id="MDEwOlJlcG9zaXRvcnkxNzA4ODYyNzY="
        data-category="General"
        data-category-id="DIC_kwDOCi-EhM4CX3Pe"
        data-mapping="og:title"
        data-strict="0"
        data-reactions-enabled="1"
        data-emit-metadata="0"
        data-input-position="top"
        data-theme="noborder_light"
        data-lang="en"
        crossorigin="anonymous"
        async>
</script>
```

Context: #1348

**Comment by niccokunzmann** [comment]
If you have a look at the commit above, you can see that I removed the origins value.
It seems that this value was causing the redirect.

It works now:

![grafik](https://github.com/giscus/giscus/assets/564768/05d66a10-f1c1-473e-931d-01b8b991db15)

So, redirecting when the origins value is the problem, is misleading. Would it be possible to display a page saying:

> This URL is forbidden to comment on the repository niccokunzmann/open-web-calendar because the giscus.json file does not contain "..." in the 'origins' list.

**Comment by giggio** [comment]
Agreed, this is misleading. I just fell for it, there should be a better response to help the user find the real problem.

**Comment by yunxi177** [comment]
I also encountered a similar issue. I previously set the origins in the giscus.json file in my comment repository, but because it was set so long ago, I forgot to update the origins in time when I changed my domain name. I think there should be a better way to handle this kind of problem. For example, adding a prompt in https://github.com/orgs/giscus/discussions/1298 would help resolve the issue more quickly.

**Comment by coycatrett** [comment]
I think I am having a similar issue: https://github.com/giscus/giscus/issues/1684. Removing the origin values hasn't worked, though.

Update: I resolved my issue.
